indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/Attestation.kt

```diff
@@ -40,18 +40,22 @@ data class IosLegacyHomebrewAttestation(
     @Serializable(with=ByteArrayBase64UrlSerializer::class)
     val attestation: ByteArray,
     @Serializable(with=ByteArrayBase64UrlSerializer::class)
+    val clientData: ByteArray,
```

Comment:
does the legacy format actually store its client data? i don't believe this was the case before...?

Reply:
the clientdata is the encoded public key. then again. we can just remove this, because Supreme should only issue in the new format and WARDEN does not require a wire format for the legacy attestation. So, I think this should actually be removed.

Reply:
any veto against removing it? @iaik-jheher

Reply:
if we don't need it, we don't need it. make it so.

Reply:
also removed from the docs. WARDEN supporting the new attestation scheme will be released before a new signum version